marshmallow_dataclass/__init__.py

```diff
@@ -122,7 +122,7 @@ def dataclass(
 # _cls should never be specified by keyword, so start it with an
 # underscore.  The presence of _cls is used to detect if this
 # decorator is being called with parameters or not.
-@dataclass_transform()
+@dataclass_transform(field_specifiers=(dataclasses.field,))
```

Comment:
According to [PEP 681], this should be
```
field_specifiers=(dataclasses,Field, dataclasses.field)
```

[pep 681]: https://peps.python.org/pep-0681/#dataclass-transform-parameters:~:text=The%20standard%20dataclass%20behavior%20supports%20only%20one%20type%20of%20field%20specifier%20called%20Field%20plus%20a%20helper%20function%20(field)%20that%20instantiates%20this%20class%2C%20so%20if%20we%20were%20describing%20the%20stdlib%20dataclass%20behavior%2C%20we%20would%20provide%20the%20tuple%20argument%20(dataclasses.Field%2C%20dataclasses.field).

Reply:
Good catch !